### DISCUSSION TITLE
[Q] Can I use the "Allow Users to Edit Range"?

### DISCUSSION THREAD

**Discussion by sdg9670** [exceljs/exceljs#2474] [Q&A]
## 💬 Questions and Help

![image](https://user-images.githubusercontent.com/40787013/82293574-bd42fb00-99e7-11ea-92d9-35f0f6a1b502.png)

 Can I use the "Allow Users to Edit Range" in exceljs?

**Comment by nikhilism15** [comment]
I am also looking for this functionality, but all I ever find is how to protect the entire sheet at once.

**Comment by RankesGitHub** [comment]
i am very wanted this

**Comment by xlboy** [comment]
i am very wanted this

**Comment by ilan-tentser** [comment]
I think this feature is very important and would like also to have it
